mungegithub/mungers/submit-queue_test.go

```diff
@@ -845,6 +846,21 @@ func TestSubmitQueue(t *testing.T) {
 			state:           "pending",
 		},
 		{
+			name:             "Fail because kind/blocker label is required but missing",
+			pr:               ValidPR(),
+			issue:            LGTMApprovedIssue(),
+			additionalLabels: []string{"kind/blocker"},
+			events:           NewLGTMEvents(),
+			commits:          Commits(), // Modified at time.Unix(7), 8, and 9
+			ciStatus:         SuccessStatus(),
+			lastBuildNumber:  LastBuildNumber(),
+			gcsResult:        SuccessGCS(),
+			retest1Pass:      true,
+			retest2Pass:      true,
+			reason:           noAdditionalLabelMessage("kind/blocker"),
+			state:            "pending",
+		},
+		{
```

Comment:
Please add a test case for the scenario in which a PR has the required additional labels.

Reply:
good call, added one